source/accessible-media/subtitles.html.erb.md

```diff
@@ -15,25 +14,11 @@ contributors:
 support: Developed with support from the <a href="https://www.w3.org/WAI/WCAGTA/">U.S. Access Board, WCAG TA Project</a>
 ---
 
-## Subtitle basics
-
 Subtitles (also known as foreign-language subtitles) are, from a
 technical perspective, similar to captions: they are text (or sometimes
 images of text) presented through markup that contains timing
 information and positioning codes. They can be created using the same
-authoring tools as captions, and they can also be delivered using the
-same basic timed-text markup formats:
-
--   [Timed Text Markup Language
-    (TTML)](https://www.w3.org/TR/ttaf1-dfxp/)
--   [Web Video Text Tracks (WebVTT)](https://w3c.github.io/webvtt/)
--   [SRT](https://matroska.org/technical/specs/subtitles/srt.html)
-
-And, as with captions, they can be open or closed:
-
--   **Closed subtitles** can be hidden and revealed by users, typically
-    by operating a button or menu on the player's control bar.
--   **Open subtitles** are visible to everyone and cannot be turned off.
+authoring tools and [formats as captions](formats.html). They can be provided in an [open or closed](index.html#open-closed) way.
 
 In instances where subtitles cannot be presented using true text, they
```

Comment:
This whole paragraph feels like a distraction:

> In instances where subtitles cannot be presented using true text, they
> can instead be presented as bitmaps: in other words, pictures of text or
> other symbols. The bitmaps themselves can still be transmitted using
> markup, though, by using [TTML Profiles for Internet Media Subtitles and
> Captions 1.0 (IMSC1)](https://www.w3.org/TR/ttml-imsc1/). Keep in mind that bitmap captions cannot be customized, which may diminish their usefulness to viewers with certain types of visual impairments.  Learn more about [why caption customization is important](production-captions.html#custom_cc).

1. Doesn’t this apply for captions as well?
2. Are bitmap subtitles a thing on the web?
3. Do we want to encourage people to use them or rather just show them the most optimal way?